src/components/10-atoms/input-text/index.js

```diff
@@ -22,7 +22,7 @@ class AXAInputText extends AXAPopupMixin(NoShadowDOM) {
       label: { type: String },
       required: { type: Boolean },
       placeholder: { type: String },
-      value: { type: String, defaultValue: undefined }, // proper default for controlled-mode under React
+      value: { type: String, reflect: true, defaultValue: undefined }, // proper default for controlled-mode under React
```

Comment:
Why does it have to reflect? this can be seen as major change. Also this needs to be tested deeply in react

Reply:
My bad